includes/admin/class-wc-admin-settings.php

```diff
@@ -622,6 +615,44 @@ class="<?php echo esc_attr( isset( $value['class'] ) ? $value['class'] : '' ); ?
 						<?php
 						break;
 
+					// Days/months/years selector.
+					case 'relative_date_selector':
+						$periods      = array(
+							'days'   => __( 'Day(s)', 'woocommerce' ),
+							'weeks'  => __( 'Week(s)', 'woocommerce' ),
+							'months' => __( 'Month(s)', 'woocommerce' ),
+							'years'  => __( 'Year(s)', 'woocommerce' ),
+						);
+						$option_value = wc_parse_relative_date_option( self::get_option( $value['id'], $value['default'] ) );
+						?>
+						<tr valign="top">
+							<th scope="row" class="titledesc">
+								<label for="<?php echo esc_attr( $value['id'] ); ?>"><?php echo esc_html( $value['title'] ); ?> <?php echo $tooltip_html; // WPCS: XSS ok. ?></label>
+							</th>
+							<td class="forminp">
+							<input
+									name="<?php echo esc_attr( $value['id'] ); ?>[number]"
+									id="<?php echo esc_attr( $value['id'] ); ?>"
+									type="number"
+									style="width: 80px;"
+									value="<?php echo esc_attr( $option_value['number'] ); ?>"
+									class="<?php echo esc_attr( $value['class'] ); ?>"
+									placeholder="<?php echo esc_attr( $value['placeholder'] ); ?>"
+									step="1"
```

Comment:
This should have a `min` of `0`, otherwise you can set it to `-5` and it becomes `5` when saved, which is sort-of confusing.

Reply:
Min of 1, or empty. Not 0.